tests/unit/emulatemb.cxx

```diff
@@ -483,5 +748,16 @@ int main(int argc, char** argv)
   testDragAndTimeout();
   testDragAndRelease();
 
+  testMBEMKeyPress(false);
+  testMBEMKeyPress(true);
```

Comment:
Feel free to be more verbose in the naming in the test as it should be easy to see here what this tests.